frontend/apps/reader/modules/readergesture.lua

```diff
@@ -101,6 +129,40 @@ function ReaderGesture:buildMenu(ges, default)
     return return_menu
 end
 
+local multiswipes = {
+    "west east",
+    "east west",
+    "north south",
+    "south north",
+    "south west",
+    "south east",
+    "east south",
+    "west south",
+}
+local custom_multiswipes_path = DataStorage:getSettingsDir().."/multiswipes"
```

Comment:
I see why, but it feels strange the "path" would not include the suffix.

Reply:
I suppose I could also do strsub or something in the util.pathExists check but it seems less transparent to me.